specification/cognitiveservices/data-plane/AzureOpenAI/authoring/stable/2022-12-01/azureopenai.json

```diff
@@ -0,0 +1,1924 @@
+              "$ref": "#/definitions/Model"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Getting a base model.": {
+            "$ref": "./examples/get_model_base.json"
+          },
+          "Getting a fine-tuned model.": {
+            "$ref": "./examples/get_model_finetune.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Capabilities": {
+      "title": "Capabilities",
+      "description": "The capabilities of a base or fine tune model.",
+      "required": [
+        "completion",
+        "embeddings",
+        "fine_tune",
+        "inference",
+        "scale_types"
+      ],
+      "type": "object",
+      "properties": {
+        "fine_tune": {
+          "description": "A value indicating whether a model can be used for fine tuning.",
+          "type": "boolean"
+        },
+        "inference": {
+          "description": "A value indicating whether a model can be deployed.",
+          "type": "boolean"
+        },
+        "completion": {
+          "description": "A value indicating whether a model supports completion.",
+          "type": "boolean"
+        },
+        "embeddings": {
+          "description": "A value indicating whether a model supports embeddings.",
+          "type": "boolean"
+        },
+        "scale_types": {
+          "description": "The supported scale types for deployments of this model.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScaleType"
+          }
+        }
+      }
+    },
+    "Deployment": {
+      "title": "Deployment",
+      "description": "Deployments manage the reserved quota for Azure OpenAI models and make them available for inference requests.",
+      "required": [
+        "model",
+        "scale_settings"
+      ],
+      "type": "object",
+      "properties": {
+        "object": {
+          "$ref": "#/definitions/TypeDiscriminator"
+        },
+        "status": {
+          "$ref": "#/definitions/State"
+        },
+        "created_at": {
+          "format": "unixtime",
+          "description": "A timestamp when this job or item was created (in unix epochs).",
+          "type": "integer",
+          "readOnly": true
+        },
+        "updated_at": {
+          "format": "unixtime",
+          "description": "A timestamp when this job or item was modified last (in unix epochs).",
+          "type": "integer",
+          "readOnly": true
+        },
+        "id": {
+          "description": "The identity of this item.",
+          "type": "string",
+          "readOnly": true
+        },
+        "model": {
+          "description": "The OpenAI model to deploy. Can be a base model or a fine tune.",
+          "minLength": 1,
+          "type": "string"
+        },
```

Comment:
nit, If possible, be more specific on what's the identifier of model, is it the "model-id" in "/models/{model-id}" API?

same for "model" under "DeploymentUpdate"